pkg/promlib/converter/testdata/prom-infos-frame.jsonc

```diff
@@ -1,10 +1,10 @@
 //  🌟 This was machine generated.  Do not edit. 🌟
 //  
 //  Frame[0] {
-//      "type": "timeseries-multi",
+//      "type": "numeric-multi",
```

Comment:
I see that:
> 🌟 This was machine generated. Do not edit. 🌟

So I'd expect it to be right. Yet I'm surprised - I feel like this should be a `timeseries-multi` still. It still has both a Time and a Value column as far as I can tell (line 26 below). Can you explain how this change was introduced? Thanks 🙏

Reply:
In the test files we weren't passing the `Dataplane: true` in options. That's why it was `timeseries-multi`. After removal it was always true and we got this update in the golden files. So the code was always working as dataplane: true but not just in test files. This is the outcome. I'm open to discuss this in a separate issue though.

Reply:
Got it, thanks ❤️ 

I've convinced myself that there's no regression in the app here (which of course you know, but I'm still finding my way around this code). It was `numeric-multi` before and after this change, for an `Instant` query (on an `_info` metric, in my case).

So yes, I will approve :-D

@kylebrandt why does https://grafana.com/developers/dataplane/numeric say:
> Numeric Kinds are generally similar to their corresponding time series type, except that their value is a single number, instead of a series of (time, numeric value). So the value of each metric is a single number like 1, 2.3, or NaN

But I see time value present when I query, in the response?

![Uploading image.png…]()

Reply:
> But I see the Time column and values present when I query, in the response?

In this case time would be "remainder data". For instant query, we know the response will not be a series data datapoints but only a single datapoint. 

So while it could be considered time series data that happens to have only a single observation, I think the more important property is that we know the data is already "Reduced" and can be used with alerting or a viz like single stat without more reduction.